### ISSUE TITLE
Realignment and break points

### ISSUE THREAD

**Issue by mmnikolic** [dellytools/delly#354]
Hi @tobiasrausch ! I have a question about "SV genotyping where reads are realigned to the REF and ALT allele.". (originally posted in https://github.com/dellytools/delly/issues/273#issuecomment-1027697099).

Can this change the break-point position? Or is the break-point position already established in the first step (SV discovery (BAM is taken as is)) where no realignment is applied?

I am curious if the realignment could cause discrepancy in visualising the reads from the BAM and the break points.

Thank you!

**Comment by mmnikolic** [comment]
Thank you @tobiasrausch! What I am curious about specifically is that for some breakpoints, soft-clipped part of supporting reads are not exactly aligned with those breakpoints (breakpoint can be outside of the soft-clipped part or inside, not on the end of soft-clipped part). Is this expected? Does it relate to CONSENSUS sequence?

**Comment by tobiasrausch** [comment]
Yes, delly computes a consensus to reference alignment using dynamic programming. If you compile from source you can also output this alignment to stderr:

[Consensus to reference alignment code](https://github.com/dellytools/delly/blob/main/src/split.h?plain=1#L664-L678)

**Comment by milonikolic** [comment]
@tobiasrausch Thank you! This explains a lot!
